web/elm/Dream/New/State.elm

```diff
@@ -0,0 +1,28 @@
+module Dream.New.State exposing (init, update, new)
+
+import Dream.Types exposing (Msg(..))
+import Dream.New.Types exposing (Dream)
+import Dream.New.Store exposing (create)
+
+new : Dream
+new =
+  { content = "" }
+
+init : ( Dream, Cmd Msg )
+init =
+  ( new, Cmd.none )
+
+update : Msg -> Dream -> ( Dream, Cmd Msg )
+update msg dream =
+  case msg of
+    NoOp ->
+      ( dream, Cmd.none )
+    ChangeContent content ->
+      ( { dream | content = content }, Cmd.none )
+    Fail _ ->
+      -- disregard error; do nothing
+      ( dream, Cmd.none )
+    CreateDone newDream ->
+      init
+    Create ->
+      ( new, Dream.New.Store.create dream.content)
```

Comment:
possible to store the `create` call in a variable? returning a function call directly from another class seems like a general code smell to me.

Sorry if my terminology is completely off here.